packages/stryker/src/stryker-cli.ts

```diff
@@ -14,6 +15,7 @@ program
   .usage('<command> [options] [stryker.conf.js]')
   .description(`Possible commands: 
     run: Run mutation testing
+    init: Initalize Stryker for your project
```

Comment:
The stryker-cli (globally installed) can be used with `stryker init` to install Stryker. If you've run that command, will you start the questionaire? I would expect that behavior